### PR TITLE
USHIFT-2992: Update MicroShift's metal host OS to rhel9.4

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-metal-bootc-test	pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-metal-periodic-test pull-ci-openshift-microshift-main-microshift-metal-tests-arm pull-ci-openshift-microshift-release-4.15-metal-periodic-test pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-metal-bootc-test pull-ci-openshift-microshift-main-metal-periodic-test pull-ci-openshift-microshift-main-microshift-metal-tests-arm pull-ci-openshift-microshift-release-4.15-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-metal-bootc-test pull-ci-openshift-microshift-main-metal-periodic-test pull-ci-openshift-microshift-main-microshift-metal-tests-arm pull-ci-openshift-microshift-release-4.15-microshift-metal-tests

File: ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh [hunk 1]
```diff
@@ -15,8 +15,10 @@ trap 'save_stack_events_to_shared' EXIT TERM INT
 declare -A ami_map=(
   [us-west-2,x86_64,rhel-9.2]=ami-0d5b3039c1132e1b2
   [us-west-2,x86_64,rhel-9.3]=ami-04b4d3355a2e2a403
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh:17]: Can we also update the 9.2 and 9.3 images? This should speed up the CI by installing less packages on `dnf upgrade all`

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-metal-bootc-test pull-ci-openshift-microshift-main-metal-periodic-test pull-ci-openshift-microshift-main-microshift-metal-tests-arm pull-ci-openshift-microshift-release-4.15-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-microshift-metal-tests-arm

**Comment by pmtk** [comment]
Depends on https://github.com/openshift/microshift/pull/3251
/hold

**Review by pacevedom — APPROVED** [review]
/lgtm

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-metal-bootc-test periodic-ci-openshift-microshift-main-ocp-metal-nightly-arm pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus-arm pull-ci-openshift-microshift-release-4.15-microshift-metal-tests

**Comment by pmtk** [comment]
/pj-rehearse periodic-ci-openshift-microshift-main-ocp-metal-nightly-arm pull-ci-openshift-microshift-main-microshift-metal-tests pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus pull-ci-openshift-microshift-main-ocp-conformance-rhel-eus-arm pull-ci-openshift-microshift-release-4.15-microshift-metal-tests

**Pull request merged**